### PR TITLE
La Grande Fusion Notes - History Editing

### PR DESCRIPTION
**Pull request by jtmcdole**
AKA "Dude Where's My Files?"

Document the history editing that will occur during the monorepo merge. `flutter/engine` will remain the source of truth of revisions older than the merge.

### CODE REVIEW

File: docs/monorepo/history_strategy.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# La Grande Fusion with History
```
Comment (zanderso) [inline docs/monorepo/history_strategy.md:1]: `La Grade Fusion` is a great code name, but probably in docs that are going to live for a while, we should use more boring terms =) Maybe: "Retaining history when merging the engine and framework repos".

File: docs/monorepo/history_strategy.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+**Goal**: retain valuable history in a way that is consumable in the final merge layout.
+
```
Comment (zanderso) [inline docs/monorepo/history_strategy.md:4]: Some more intro could be helpful here for someone reading this in 6-12 months. Maybe something like, "These are the steps we followed to selectively retain flutter/engine repo history when merging the engine repo into the flutter/flutter repo. The idea was to retain as much useful history as possible without blowing up the footprint of the framework's .git folder."

**Review by zanderso — APPROVED** [review]
Thanks!

File: docs/monorepo/history_strategy.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+--path 'testing/scenario_app/android/reports' \
+--path 'impeller/third_party' \
+--path 'mojo/public/third_party' \
+--path 'tests/data' \
+--path 'tests/fast' \
+--path 'tests/framework' \
+--path 'travis' \
+--path 'mojo' \
+--path 'sky/sdk' \
+--path 'sky/engine' \
```
[the remark this answers is not in the document]
Comment (jtmcdole) [reply docs/monorepo/history_strategy.md:140]: Yep. I have a local copy and I caught this on Friday but didn't upload a new PR

**Pull request merged**